python/lsst/daf/butler/formatters/fitsExposureFormatter.py

```diff
@@ -103,6 +103,18 @@ def readComponent(self, component, parameters=None):
                         'image': ('readImage', True),
                         'mask': ('readMask', True),
                         'variance': ('readVariance', True),
+                        'photoCalib': ('readPhotoCalib', False),
+                        'bbox': ('readBBox', True),
+                        'xy0': ('readXY0', True),
+                        'metadata': ('readMetadata', False),
+                        'filter': ('readFilter', False),
+                        'polygon': ('readValidPolygon', False),
+                        'appCorr': ('readApCorrMap', False),
```

Comment:
This should be `apCorrMap` not `appCorr`

Reply:
OK.  I'll make that change.